ui/src/components/TestRunner/index.tsx

```diff
@@ -305,31 +301,54 @@ export const TestRunner = () => {
           setSupportedSeps={setSupportedSeps}
         />
         {supportedSeps.length !== 0 && (
-          <Select
-            id="seps"
-            label="sep"
-            onChange={(e) => handleSepChange(e.target.value)}
-          >
-            <option></option>
-            {supportedSeps.map((sepNum) => (
-              <option key={sepNum} value={String(sepNum)}>
-                SEP-{sepNum}
-              </option>
-            ))}
-          </Select>
+          <FieldWrapper>
+            <>
```

Comment:
you actually shouldn't need this `<>` as `FieldWrapper` is already wrapping everything inside